src/components/dashboard/main-content/course-enrollments/CourseSection.jsx

```diff
@@ -126,6 +127,9 @@ class CourseSection extends React.Component {
         </UpgradeableCourseEnrollmentContextProvider>
       );
     }
+    if (courseRun.isCancelledAssignment) {
+      return null;
+    }
```

Comment:
[suggestion] This implies we're still including/iterating over canceled assignments. I wonder if it'd make sense to do filtering on the list of assignments further up in the logic to exclude the already-dismissed canceled assignments as opposed to here more in the presentation layer.

For example, perhaps the `getActiveAssignments` function could update its `activeAssignments` logic to account for whether the canceled assignment(s) has already been dismissed or not (based on the last dismissed cancelations timestamp or the uuid matching).

That way, when other logic relies on the `hasActiveAssignments` returned by the function, the boolean would be more reflective of what would be shown in the UI, e.g. if only already-dismissed cancelations are present, `hasActiveAssignments=false`.